_includes/02-image.md

```diff
@@ -0,0 +1 @@
+Replace this with an image, like your photo. Ensure you include some alt-text.
```

Comment:
![Hints of India Outfit Glamor Shot](https://user-images.githubusercontent.com/41228164/43714052-98bce494-9930-11e8-8a3e-060555990611.jpg)